css/properties.schema.json

```diff
@@ -7,6 +7,7 @@
       "items": {
         "type": "string",
         "property-reference": {
+          "comment": "That's a non-standart extension to JSON Schema validator. It tests value is an existing key in root object (i.e. defined property). See test/validate-schema.js for extension implementation details",
```

Comment:
Better:
"property-reference is an extension to the JSON schema validator. Here it jumps to the root level of the hierarchy and tests if a value is an existing key there (i.e a defined property). See test/validate-schema.js for implementation details."